### PR TITLE
Sequencer testing

### PR DESCRIPTION
**Pull request by bpeabody**
See card: https://github.com/twosigma/git-meta/projects/2#card-8117137

### CODE REVIEW

File: node/lib/util/shorthand_parser_util.js [hunk 1]
```diff
@@ -79,6 +83,10 @@ const RepoASTUtil  = require("../util/repo_ast_util");
  * rebase         = E<head name>,<original commit id>,<onto commit id>
  * merge          = M<message>,<original head>,<merge head>
  * cherry-pick    = P<original head>,<picked commit>
+ * sequencer      = Q<sequencer type>' '<commit and ref>' '<commit and ref>' '
```
Comment (novalis) [inline node/lib/util/shorthand_parser_util.js]: Can you describe what the format of commit-and-ref is as well as the semantics of each value?  As in &lt;original-head-ref:sha&gt;

Comment (bpeabody) [reply node/lib/util/shorthand_parser_util.js]: It's just a couple of lines under that one:

```
 * sequencer      = Q<sequencer type>' '<commit and ref>' '<commit and ref>' '
 *                  <number>' '<commit>(','<commit>)*
 * sequencer type = C | M | R
 * commit and ref = commit':'[<name>]
```

Comment (bpeabody) [reply node/lib/util/shorthand_parser_util.js]: I've got an example a little further down too:

```
 * QR 1:refs/heads/master 8: 1 3,5,1
 *                              -- A sequencer is in progress that is a 
 *                              -- rebase.  When the rebase started, HEAD
 *                              -- was on `master`.  
```

Comment (novalis) [reply node/lib/util/shorthand_parser_util.js]: Oh, the format is -- that makes sense. But the semantics are still obscure to me.

**Pull request merged**